build/charts/theia/provisioning/dashboards/flow_records_dashboard.json

```diff
@@ -818,6 +818,168 @@
                 "value": 181
               }
             ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "flowEndReason"
+            },
+            "properties": [
+              {
+                "id": "mappings",
+                "value": [
+                  {
+                    "options": {
+                      "1": {
+                        "index": 0,
+                        "text": "Idle Timeout"
+                      },
+                      "2": {
+                        "index": 1,
+                        "text": "Active Timeout"
+                      },
+                      "3": {
+                        "index": 2,
+                        "text": "End Of Flow"
+                      }
+                    },
+                    "type": "value"
+                  }
+                ]
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "timeInserted"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "time when flow inserted to ClickHouse"
```

Comment:
Shall we come up with shorter names? e.g. "database insertion time"

as we edit the column name, we might also want to adjust the column width to show the entire column name? like what we do in L810-819. If you drag the borderline of any column to adjust its width, the override property will generate automatically.

Reply:
Good suggestion. I've updated it. Thanks.